src/common/__tests__/dateHelpers.test.js

```diff
@@ -53,4 +53,39 @@ describe('DateHelpers', () => {
     expect(getRangedMonthDateTime('april')).toMatchSnapshot('get a specific month by name');
     expect(getRangedMonthDateTime(3)).toMatchSnapshot('get a specific month by number');
   });
+
+  it('setEndOfDay should change date to end of day', () => {
+    const startDate = new Date(Date.UTC(2020, 7, 15, 3, 6, 8, 3));
+    const expectedDate = new Date(Date.UTC(2020, 7, 15, 23, 59, 59, 999));
+
+    expect(dateHelpers.setEndOfDay(startDate).getTime()).toEqual(expectedDate.getTime());
+  });
+
+  it('setStartOfDay should change date to start of day', () => {
+    const startDate = new Date(Date.UTC(2020, 7, 15, 3, 6, 8, 3));
+    const expectedDate = new Date(Date.UTC(2020, 7, 15, 0, 0, 0, 0));
+
+    expect(dateHelpers.setStartOfDay(startDate).getTime()).toEqual(expectedDate.getTime());
+  });
+
+  it('setEndOfMonth should change date to end of Month', () => {
+    const startDate = new Date(Date.UTC(2020, 7, 15, 3, 6, 8, 3));
+    const expectedDate = new Date(Date.UTC(2020, 7, 31, 23, 59, 59, 999));
+
+    expect(dateHelpers.setEndOfMonth(startDate).getTime()).toEqual(expectedDate.getTime());
+  });
+
+  it('setRangedDateTime with weeks, should start the weeks on Sunday.', () => {
```

Comment:
add in the test for the default option changes noted above

optional
consider merging this with the existing `setRangedDateTime` test around line `32`

Reply:
merged and added.